test/ex_webrtc/peer_connection_test.exs

```diff
@@ -37,6 +37,9 @@ defmodule ExWebRTC.PeerConnectionTest do
 
     assert_receive {:ex_webrtc, ^pc, {:track, %MediaStreamTrack{kind: :audio}}}
 
+    {:ok, answer} = PeerConnection.create_answer(pc)
```

Comment:
Would be nice to add a test that should fail in the previous implementation but should pass in the current one